src/server.c

```diff
@@ -5248,7 +5248,7 @@ NULL
     addReplyHelp(c, help);
 }
 
-/* Convert an amount of bytes into a human readable string in the form
```

Comment:
something like that is ok.
please check it from https://encyclopedia.thefreedictionary.com/human-readable